### PR TITLE
Update build-first-app-overview.md

### PR DESCRIPTION
**Pull request by karthigb**
Made a clearer call to action on the prerequisites page.

### CODE REVIEW

File: msteams-platform/build-your-first-app/build-first-app-overview.md [hunk 1]
```diff
@@ -51,11 +51,13 @@ You need a Teams account that allows custom app sideloading. (Your account may a
     1. Look for an option to **Upload a custom app**.
 
     :::image type="content" source="../assets/images/build-your-first-app/upload-custom-app-closeup.png" alt-text="Illustration showing where in Teams you can upload a custom app.":::
+    
+    If you don't see the button, you don't have permission to install custom apps in your organization. Read the next section to learn how to acquire your own free test subscription.
```
Comment (heath-hamilton) [inline msteams-platform/build-your-first-app/build-first-app-overview.md:55]:
```suggestion
    If you don't see the button, you don't have permission to upload custom apps in your org. You can get this feature by signing up for a free Microsoft 365 developer subscription.
```
@karthigb Changed the wording slightly to align with the dev program verbiage. Is this OK?

File: msteams-platform/build-your-first-app/build-first-app-overview.md [hunk 2]
```diff
@@ -51,11 +51,13 @@ You need a Teams account that allows custom app sideloading. (Your account may a
...
 
 <!-- markdownlint-disable MD033 -->
 <details>
 
-<summary><b>Select here</b> if you can't see the sideload option or don't have a Teams account.</summary>
+<summary><b>Get a free Microsoft Teams subscription</b></summary>
```
Comment (heath-hamilton) [inline msteams-platform/build-your-first-app/build-first-app-overview.md:60]:
```suggestion
<summary><b>Get your free Microsoft 365 developer subscription</b></summary>
```
@karthigb Changed again to align with the dev program site.

**Review by heath-hamilton — COMMENTED** [review]
@karthigb Made a couple minor tweaks. Hope these help--thanks in advance!

**Comment by heath-hamilton** [comment]
@surbhigupta12 This branch is ready to merge. Thank you!

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR

**Pull request merged**